data/moves.js

```diff
@@ -3413,6 +3413,11 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {protect: 1, reflectable: 1, mirror: 1, authentic: 1},
 		volatileStatus: 'disable',
+		onTryHit: function (target) {
+			if (!target.lastMove || target.lastMove.isZ) {
```

Comment:
Couldn't this check just be in the `onStart` function, which already checks `.lastMove` anyway? At the very least, it's redundant to check that twice...